Neos.Flow/Classes/Validation/Validator/TextValidator.php

```diff
@@ -36,7 +36,7 @@ class TextValidator extends AbstractValidator
      */
     protected function isValid($value)
     {
-        if ($value !== filter_var($value, FILTER_SANITIZE_STRING, FILTER_FLAG_NO_ENCODE_QUOTES)) {
+        if ($value !== strip_tags((string)$value)) {
```

Comment:
FILTER_FLAG_NO_ENCODE_QUOTES should be replaced with https://www.php.net/manual/de/function.htmlspecialchars.php - so should we do that as well here?

Reply:
Tried that first, but for some reason a test failed then. As per the method description "text without any XML tags" is to be considered valid, so I figured `strip_tags()` should od the job – and the tests pass.